fdbserver/BlobWorker.actor.cpp

```diff
@@ -1028,6 +1028,14 @@ ACTOR Future<BlobFileIndex> writeEmptyDeltaFile(Reference<BlobWorkerData> bwData
 			if (!dfValue.present()) {
 				// Only check if not seen yet. If we get commit unknown result and then retry, we'd see our own delete
 				wait(store(dfValue, tr->get(oldDFKey)));
+				if (!dfValue.present()) {
```

Comment:
since we have assertion for dfValue.present() in line 1039, we should not need to check if dfValue is present or not.

Reply:
Yeah i specifically just wanted to trace an event with details if the assert was going to fail before the assert actually fails, for context